pkg/apis/core/validation/validation_test.go

```diff
@@ -20530,7 +20609,7 @@ func TestValidatePVSecretReference(t *testing.T) {
 			name:          "invalid secret ref name",
 			args:          args{&core.SecretReference{Name: "$%^&*#", Namespace: "default"}, rootFld},
 			expectError:   true,
-			expectedError: "name.name: Invalid value: \"$%^&*#\": " + dnsLabelErrMsg,
+			expectedError: "name: Invalid value: \"$%^&*#\": " + dnsLabelErrMsg,
```

Comment:
revert this change, if we're saying the root field is "name", then the name attribute of the ref is `name.name`

Reply:
sure, done.